From 2fc539d16603bcaff9f735741c32e04d736d122b Mon Sep 17 00:00:00 2001
From: Manuel Buil <mbuil@suse.com>
Date: Wed, 17 Feb 2021 13:17:09 +0100
Subject: [PATCH] fix modebuild

---
 scripts/build.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/build.sh b/scripts/build.sh
index c5ec53f..fd1d564 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -29,5 +29,5 @@ export GOBIN=${PWD}/bin
 export CGO_ENABLED=0
 export GO15VENDOREXPERIMENT=1
 
-go install -ldflags "-s -w" -buildmode=pie -tags no_openssl "$@" ${REPO_PATH}/cmd/installer
-go install -ldflags "-s -w" -buildmode=pie -tags no_openssl "$@" ${REPO_PATH}/cmd/webhook
+go install -ldflags "-s -w" -tags no_openssl "$@" ${REPO_PATH}/cmd/installer
+go install -ldflags "-s -w" -tags no_openssl "$@" ${REPO_PATH}/cmd/webhook
-- 
2.26.2

